src/app/games/lines-of-action/tests/LinesOfActionMove.spec.ts

```diff
@@ -19,7 +19,7 @@ describe('LinesOfActionMove', () => {
         expect(LinesOfActionMove.of(new Coord(7, 7), new Coord(9, 9))).toEqual(MGPFallible.failure('end coord is not in range'));
     });
     it('should not create a move with an invalid direction', () => {
-        expect(LinesOfActionMove.of(new Coord(3, 3), new Coord(4, 5))).toEqual(MGPFallible.failure('Invalid delta for direction: 1, 2'));
+        expect(LinesOfActionMove.of(new Coord(3, 3), new Coord(4, 5))).toEqual(MGPFallible.failure('Non linear move are not allowed: 1, 2'));
```

Comment:
euuh j'suis pas certain que ça soit mieux ça :D

Reply:
bah, y a que des mots que les gens normaux connaissent, là on mentionne bien que ça parle du mouvement et y a pas besoin de savoir qu'on parle de deltas.
Tu dirais quoi ? Quelquechose de plus clair genre "ce mouvement n'est ni orthogonal ni diagonal (dx, dy)" ?

Reply:
Oui, et plus formuler d'une façon user-friendly. « You did not move in a straight line! You can only move orthogonally or diagonally. » Et peut-être pas mentionner le delta ? Puis aussi, localizer cela ? Vu que c'est pas infréquent qu'il arrive dans un toast!

Reply:
J'ferais bien une enquête UX mais Henri c'est un ancien étudiant en math, Mathieu c'est un dev, Hadrien est probablement aussi plus numérophile que la moyenne, si je leur demande il y a une petite chance qu'ils soient favorable au delta sans être représentatifs :p

Reply:
Je crois que « linéaire » c'est un mot qui fait peur si t'es pas un minimum matheux/technique, mais j'te laisse libre choix de reformuler, tu as ma proposition ! Et pas de "Mouvement illégal: 1, 2" où la personne sait pas d'où viennent ces nombres

Reply:
et un "You must move in a straight line! You can only move orthogonally or diagonally!" qui va bien!